src/app/support-training/support-training.component.html

```diff
@@ -14,12 +14,9 @@ <h2>Unmet Hours Website</h2>
         StackExchange Q&A websites. Users can ask and answer questions and “vote” questions and answers up or down.
         Users can also subscribe to interesting topics and receive email notifications when new questions about the
         topic are asked—members of the EnergyPlus team subscribe to the tag ‘energyplus’.</p>
-      <h2>Real-Time Email and Phone Support</h2>
-      <p>These <a href="https://buildingenergysoftwaretools.com/?capabilities=Support+Services&keys=EnergyPlus">organizations</a>
-       provide more individual email and/or phone support.</p>
-      <h2>EnergyPlus Training</h2>
-      <p>Several <a href="https://buildingenergysoftwaretools.com/?capabilities=Training+Services&keys=EnergyPlus">organizations</a>
-        provide public and/or private EnergyPlus trainings and workshops.</p>
```

Comment:
Could not provide a link to support and training individually, so I just lumped them together.  Once on that target page, it's pretty clear how to down-select for support, etc.